pkg/jvscrypto/rotation_handler_test.go

```diff
@@ -162,6 +172,15 @@ func TestDetermineActions(t *testing.T) {
 				{ActionCreateNewAndPromote, nil},
 			},
 		},
+		{
+			name: "single_key_new",
+			versions: []*kmspb.CryptoKeyVersion{
+				newEnabledKey2,
+			},
+			wantActions: []*actionTuple{
+				{ActionPromote, newEnabledKey2},
+			},
+		},
```

Comment:
Do we also want to test multiple new keys?

Reply:
Sure, will fix it in my next PR.